proto/cosmos/slashing/v1beta1/slashing.proto

```diff
@@ -32,6 +32,13 @@ message ValidatorSigningInfo {
   int64 missed_blocks_counter = 6 [(gogoproto.moretags) = "yaml:\"missed_blocks_counter\""];
 }
 
+// Stores a sliding window of the last `signed_blocks_window` blocks indicating whether the validator missed the block
+message ValidatorMissedBlockArray {
+  string address = 1;
+  // Array of length signing signed blocks window that contains whether the validator missed the block at that index
```

Comment:
how come we decided to store array of bool rather than just missed block heights?

Reply:
+1, this should be a slice of blockId right?